samples/tfm_integration/tfm_ipc/prj.conf

```diff
@@ -0,0 +1 @@
+CONFIG_BUILD_WITH_TFM=y
```

Comment:
Could you add some of the options that are added in level_1 prj.conf?
E.g. configurations for HEAP, thread options, etc. I would think they would be useful

Reply:
See https://github.com/zephyrproject-rtos/zephyr/pull/19985/commits/e078c697b48f5b0d3ec474b05eaa4d80aee7dd30